src/components/tools/Charts/LineArea/Chart.svelte

```diff
@@ -60,27 +65,36 @@
 
   $: if (Array.isArray(data) && !isEmptyData(data)) {
     noData = false;
+
     // Set X Domain
     xmin = min(data, (arr) => min(arr.values, (d) => d.date));
     xmax = max(data, (arr) => max(arr.values, (d) => d.date));
-    if (xAxis.baseValue === 0) {
-      xmin = xAxis.baseValue;
-    }
 
     // Set Y Domain
-    ymax = max(data, (arr) =>
-      max(arr.values, (d) => ("max" in d ? d.max : d.value))
-    );
-    if (typeof yAxis.baseValue === "number" && !isNaN(yAxis.baseValue)) {
-      ymin = yAxis.baseValue;
+    if (typeof yAxis.minDefault === "number" && !isNaN(yAxis.minDefault)) {
+      ymin = yAxis.minDefault;
     } else {
       ymin = min(data, (arr) =>
         min(arr.values, (d) => ("min" in d ? d.min : d.value))
       );
     }
 
-    ymin = +yAxis.tickFormat(ymin);
-    ymax = +yAxis.tickFormat(ymax);
+    ymax = max(data, (arr) =>
+      max(arr.values, (d) => ("max" in d ? d.max : d.value))
+    );
+
+    // Check if ymin & ymax are almost equal
+    const tolerance =
```

Comment:
@clhenrick you are right that the semantics of the terms `minDefault` & `maxDefault` are confusing in this case. What about this idea instead:
 - add props `domainMin` & `domainMax`. If either of these props are present they will override the domain `ymin` & `ymax` derived from data
  - add `niceMax` prop. If this prop is present it will update domain ymax if `niceMax > ymax`

Reply:
@mukhtyar good idea to look at how Vega handles this. I think using the `domainMin`, `domainMax`, and `niceMax` properties for the `yAxis` prop makes sense. How would `tolerance` be used here? Or would we abandon that approach of determining if the `ymin` and `ymax` are almost equal?

Reply:
@clhenrick yes we would get rid of the `tolerance` approach

Reply:
@mukhtyar sounds good to me.